docs/self-hosted-getting-started.md

```diff
@@ -0,0 +1,99 @@
+# Quick Start: Run a self-hosted database instance with automatic vectorization
+
+If you want to see how it works, follow these steps to setup 
+a self-hosted database instance with automatic embedding vectorization:
+
+## Pre-requisites
+- Docker installed on your machine
+- An OpenAI API key. You can create one [here](https://platform.openai.com/api-keys).
+
+## Setup
+
+1. Run the following docker-compose file to start a TimescaleDB instance and the Vectorizer worker: 
+    ```yaml
+    services:
+      db:
+        image: timescale/timescaledb-ha:cicd-024349a-arm64
+        environment:
+          POSTGRES_PASSWORD: postgres
+          OPENAI_API_KEY: your-api-key
+        ports:
+          - "5432:5432"
+        volumes:
+          - ./data:/var/lib/postgresql/data
+    
+      vectorizer:
```

Comment:
let call this `pgai-vectorizer-worker`

Reply:
I've switched this again to just vectorizer-worker since the namespace of the compose is now pgai and the container is otherwise called `pgai-pgai-vectorizer-worker`. Is that fine with you?

Reply:
I've switched this again to simply `vectorizer-worker` to avoid the container being called `pgai-pgai-vectorizer-worker` because I've also added the compose namespace (pgai). Is that fine with you?